mmv1/third_party/terraform/services/compute/resource_compute_instance_group_manager_meta.yaml.tmpl

```diff
@@ -33,11 +33,17 @@ fields:
   - field: 'standby_policy.initial_delay_sec'
   - field: 'standby_policy.mode'
   - field: 'stateful_disk.delete_rule'
+    api_field: 'stateful_policy.preserved_state.disks.delete_rule'
   - field: 'stateful_disk.device_name'
+    api_field: 'stateful_policy.preserved_state.disks.device_name'
   - field: 'stateful_external_ip.delete_rule'
+    api_field: 'stateful_policy.preserved_state.external_ips.delete_rule'
   - field: 'stateful_external_ip.interface_name'
+    api_field: 'stateful_policy.preserved_state.external_ips.interface_name'
   - field: 'stateful_internal_ip.delete_rule'
+    api_field: 'stateful_policy.preserved_state.internal_ips.delete_rule'
```

Comment:
It looks like the [API fields](https://cloud.google.com/compute/docs/reference/rest/v1/instanceGroupManagers) have non-standard capitalization - for example, `internalIPs` - will that cause any problems?